internal/provisioner/kops_utils.go

```diff
@@ -112,6 +112,44 @@ func getLoadBalancerEndpoint(ctx context.Context, namespace string, logger log.F
 	}
 }
 
+// addHelmRepo adds a new Helm repo
+func addHelmRepo(repoName, repoURL string, logger log.FieldLogger) error {
+	var cmd *exec.Cmd
+	arguments := []string{
+		repoName,
+		repoURL,
+	}
+
+	cmd = exec.Command("helm repo add", arguments...)
+
+	logger.WithFields(log.Fields{
+		"cmd":  cmd.Path,
+		"args": cmd.Args,
+	}).Info("Invoking command")
+
+	err := cmd.Run()
+	if err != nil {
```

Comment:
We could just return err here without checking it. It makes sense to keep the if check if you want to wrap the error with more info.

Reply:
Fixed ;)